views-turbo/src/main/java/io/micronaut/views/turbo/AbstractTurboRenderer.java

```diff
@@ -29,7 +29,8 @@
  * @since 3.4.0
  * @param <T> The class to be built
  */
-public abstract class AbstractTurboRenderer<T extends TemplatedBuilder<?>> {
+public abstract class AbstractTurboRenderer<T extends TemplatedBuilder<?, T>> {
```

Comment:
make the class `@Internal`. Can it be `protected`?

Reply:
Marked it internal... not sure what you mean by make it protected :-/

It can be package private...

Reply:
lets make it package private.